setup.py

```diff
@@ -107,6 +107,7 @@ def run(self):
     "jaraco.windows==3.9.2",
     "mock-ssh-server>=0.5.0",
     "moto>=1.3.14.dev55",
+    "rangehttpserver==1.2.0",
```

Comment:
@efiop @Suor this is not Apache 2 compatible I believe.

Also, I wonder what's happening with Conda. We'll need to wriate a checklist on adding new dependencies.

Reply:
sorry, false alarm ... there are two of them on Github

Reply:
@shcheklein I'm not sure our conda packages are running tests though. We've added this package only as a [tests] dependency.